core/src/com/unciv/ui/components/extensions/FormattingExtensions.kt

```diff
@@ -22,9 +22,10 @@ import java.util.SortedMap
 @Pure fun Float.toPercent() = 1 + this/100
 
 /** Convert a [resource name][this] into "Consumes [amount] $resource" string (untranslated) */
-@Pure fun String.getConsumesAmountString(amount: Int, isStockpiled: Boolean): String {
-    val uniqueString = "{Consumes [$amount] [$this]}"
-    return if (isStockpiled) "$uniqueString /${Fonts.turn}" else uniqueString
+@Pure fun String.getConsumesAmountString(amount: Int, isStockpiled: Boolean, perTurn: Boolean = true, label: String = "Consumes"): String {
+    val uniqueString = "{$label [$amount] [$this]}"
+    val perTurnString = if (perTurn) " /${Fonts.turn}" else ""
+    return if (isStockpiled) "${uniqueString}${perTurnString}" else uniqueString
```

Comment:
Practically there are only 2 options here, "Consumes" and "Costs". Costs doesn't accept a per-turn, so this should be solit into 2 functions - one for consumes and one for costs.
That way also the function name remains indicative